babel/index.babel

```diff
@@ -0,0 +1,27 @@
+// !! IMPORTANT README:
+
+// You may add additional external JS and CSS as needed to complete the project, however the current external resource MUST remain in place for the tests to work. BABEL must also be left in place. 
```

Comment:
I suggest you update your readme file based on microverse readme template. it will make your project look more professional.

Reply:
- [Optional] Great job being polite and clear. Consider using a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖😁.